Config/Sample.cfg

```diff
@@ -44,7 +44,7 @@ BaseLayerSwitchMode             : 0             # 0 : Use B-frames in the base l
 PredStructure                   : 2             # 2: Random Access
 
 IntraPeriod                     : 47            # Period of I-Frame (-1 = only first, -2 = auto) [-2 - 255]
-IntraRefreshType                : 2             # Random Accesss 1:CRA, 2:IDR (when intra_period > 0) - [1-2]
+IntraRefreshType                : 1             # Random Access 1:CRA, 2:IDR (when intra_period > 0) - [1-2]
```

Comment:
should it be 2 IDR? pls also update user guide?

Reply:
yes, updated in user guide and config file to match (default is 2=IDR)